sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -57,7 +63,15 @@ export class SearchIndexClient {
   /**
    * The API version to use when communicating with the service.
    */
-  public readonly apiVersion: string = "2020-06-30-Preview";
+  public readonly serviceVersion: string = utils.defaultServiceVersion;
+
+  /**
+   * The API version to use when communicating with the service.
+   * @deprecated use {@Link serviceVersion} instead
+   */
+  public get apiVersion(): string {
+    return this.serviceVersion;
+  }
```

Comment:
instead of making this a smart getter (which changes its public type) can we just set it when we set the other one, since they're both readonly?

Reply:
Done